math_.py

```diff
@@ -13,3 +13,14 @@ def substract(a: int, b: int) -> int:
 def list_substract(a: List, b: List) -> List:
     assert len(a)==len(b), "len(a) != len(b)"
     return [i-j for i, j in zip(a,b)]
+
```

Comment:
The function `isnum` checks if a value is an integer, but it also tries to convert strings to integers. This could lead to unexpected behavior if the string is not a valid integer. Consider using `isdigit()` instead of `int()` to check if a string is a valid integer.